app/code/Magento/Customer/Api/Data/OptionInterface.php

```diff
@@ -17,11 +17,23 @@ interface OptionInterface
     /**#@+
      * Constants for keys of data array
      */
+    const ID = 'id';
     const LABEL = 'label';
     const VALUE = 'value';
     const OPTIONS = 'options';
     /**#@-*/
 
+    /**
+     * @return int|null
```

Comment:
we can't add new methods to a an api public interface in between minor releases
we have to think of an alternative